ocf_datapipes/batch/merge_batchml.py

```diff
@@ -0,0 +1,2 @@
+from torchdata.datapipes import functional_datapipe
+from torchdata.datapipes.iter import IterDataPipe
```

Comment:
Should there be something else in here? Some sort of BatchML object? Similar to what is in power perceiver?